llvm-node.d.ts

```diff
@@ -228,6 +228,113 @@ declare namespace llvm {
     private constructor();
   }
 
+  class DIBasicType extends DIType {
+    readonly isSigned: bool;
```

Comment:
No such method exists on DIBasicType (llvm-node is only a thin wrapper around llvm methods). 

I assume it's a simplification for `getSignedness`. I would prefer exposing the enum and making the return type optional